### DISCUSSION TITLE
Su cuenta ha sido marcada.

### DISCUSSION THREAD

**Discussion by LeaBertero** [community/community#42401] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Por eso, su perfil está oculto al público. Si cree que se trata de un error, póngase en [contacto con el servicio de asistencia](https://github.com/contact) para que revisen el estado de su cuenta. (estoy tenienod problemas con esto, es temporal, alguen sabe o es para siempre? :/ que puedo hacer al respecto?)

**Comment by cehfisher** [comment]
Hello @LeaBertero - it looks like your account has been flagged. Tagging in @queenofcorgis to redirect your issue to a different discussion board, as this one is dedicated to accessibility issues and questions. Good luck!
